src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -85,6 +85,36 @@ def workaround_squeeze_node_xgboost(onnx_model: onnx.ModelProto):
     onnx_model.graph.node[target_node_id_list[0]].input.insert(1, axes_input_name)
 
 
+def assert_add_node_and_constant_in_xgboost_regressor_graph(onnx_model: onnx.ModelProto) -> bool:
```

Comment:
remove "-> bool", this function does not return anything !

Reply:
Indeed thanks

Reply:
done